CHANGES

```diff
@@ -55,6 +55,7 @@ Version 7.1.0 - In Development
       headers were installed to /usr/include.
     - Fixed a rotation order bug in tools::local_util::decompose()
       that caused it to fail to decompose some matrices.
+    - Fixed a bug where math::Quat::inverse() was not marked as const.
```

Comment:
missing contribution

Reply:
fixed (to speed this process up, can you approve pending a change like this?)